packages/platform-android/native_modules.gradle

```diff
@@ -150,7 +150,7 @@ class ReactNativeModules {
       packageImports = packageImports + packages.collect {
         "// ${it.name}\n${it.packageImportPath}"
       }.join('\n')
-      packageClassInstances = ",\n      " + packages.collect { it.packageInstance }.join(',')
+      packageClassInstances = ",\n      " + packages.collect { it.packageInstance }.join(",\n      ")
```

Comment:
this updates the generated PackageList.java to be nicely formatted around package instance invocations